backend/infrahub/core/utils.py

```diff
@@ -191,6 +191,12 @@ def convert_ip_to_binary_str(
     return ip_bin.zfill(obj.max_prefixlen)
 
 
+def collapse_ipv6_address_or_network(address_or_network: str) -> str:
+    if "/" in address_or_network:
```

Comment:
Using a try/except block instead of an if clause would probably be a better way to handle the network or address case.